roles/smallfile-bench/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
```

Comment:
again, why is "localhost" used?

Reply:
Just mentioned the reason in the above comment.

Reply:
More boilerplate.